src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCNV.java

```diff
@@ -258,13 +259,19 @@ protected void runPipeline(final JavaSparkContext ctx) {
             modeller.writeACNVModeledSegmentFile(modeledSegmentsFile);
         }
         logger.info("Final number of segments after similar-segment merging: " + modeller.getACNVModeledSegments().size());
+
         //write final model fit to file
         final File finalModeledSegmentsFile = new File(outputPrefix + "-" + FINAL_SEG_FILE_TAG + ".seg");
         modeller.writeACNVModeledSegmentFile(finalModeledSegmentsFile);
+
         //write file for GATK CNV formatted seg file
         final File finalModeledSegmentsFileAsGatkCNV = new File(outputPrefix + "-" + FINAL_SEG_FILE_TAG + "." + GATK_SEG_FILE_TAG + ".seg");
         modeller.writeModeledSegmentFile(finalModeledSegmentsFileAsGatkCNV);
 
+        // Write file for ACS- compatible output to help Broad CGA
+        final File finalACSModeledSegmentsFile = new File(outputPrefix + "-" + FINAL_SEG_FILE_TAG + "." + CGA_ACS_SEG_FILE_TAG + ".seg");
+        modeller.writeACNVModeledSegmentFileAsAllelicCapSegFile(finalACSModeledSegmentsFile);
+
```

Comment:
Sample name is still present here (and the test class) and in ExomeStandardArgumentDefinitions...did you want to completely remove it?  Shouldn't affect the eval scripts as the output prefix is used instead.